src/common/version/build.rs

```diff
@@ -25,5 +27,9 @@ fn main() -> shadow_rs::SdResult<()> {
         }
     );
     build_data::set_BUILD_TIMESTAMP();
-    shadow_rs::new()
+
+    let src_path = env::var("CARGO_WORKSPACE_DIR").or_else(|_| env::var("CARGO_MANIFEST_DIR"))?;
+    let out_path = env::var("OUT_DIR")?;
+    let _ = shadow_rs::Shadow::build_with(src_path, out_path, Default::default())?;
+    Ok(())
```

Comment:
Just to ensure that this requires version bump?

The commit message looks misleading .. We primarily want to fix this issue, while version bump is part of the requirement ..

Reply:
The `build_with` function is made pub in new `shadow-rs` version, see https://github.com/baoyachi/shadow-rs/pull/169. The fix is actually in our another repo, not here.